datafusion/functions/src/math/random.rs

```diff
@@ -83,19 +84,13 @@ fn random(args: &[ColumnarValue]) -> Result<ColumnarValue> {
 
 #[cfg(test)]
 mod test {
-    use std::sync::Arc;
-
-    use arrow::array::NullArray;
-
     use datafusion_common::cast::as_float64_array;
-    use datafusion_expr::ColumnarValue;
 
-    use crate::math::random::random;
+    use super::*;
 
     #[test]
     fn test_random_expression() {
-        let args = vec![ColumnarValue::Array(Arc::new(NullArray::new(1)))];
-        let array = random(&args)
+        let array = random(&[])
```

Comment:
remove this test, since it is already covered

https://github.com/apache/datafusion/blob/da82cece8181d5af0d38243229ecb2b4722cb4a9/datafusion/sqllogictest/test_files/expr.slt#L788-L790